Sources/SwiftRefactor/ExpandEditorPlaceholder.swift

```diff
@@ -94,18 +92,18 @@ struct ExpandSingleEditorPlaceholder: EditRefactoringProvider {
 
     let expanded: String
     if let functionType = placeholder.typeForExpansion?.as(FunctionTypeSyntax.self) {
-      let basicFormat = BasicFormat(
+      let format = ClosureLiteralFormat(
```

Comment:
Could we make the format that’s used to expand the editor placeholder an option that still defaults to `BasicFormat`? For example, when you explicitly run the refactoring to expand a placeholder to a trailing closure, I think it makes sense to have the multi-line behavior. Then we could just override it with `ClosureLiteralFormat` for SourceKit-LSP code completion expansion. That way `ClosureLiteralFormat` could also be specified in SourceKit-LSP and wouldn’t become public API of swift-syntax.